airbyte-integrations/bases/base-normalization/integration_tests/resources/test_primary_key_streams/data_input/catalog.json

```diff
@@ -36,172 +36,7 @@
       "sync_mode": "incremental",
       "cursor_field": [],
       "destination_sync_mode": "overwrite"
-    },
-    {
-      "stream": {
-        "name": "dedup_exchange_rate",
```

Comment:
Why is this test case getting deleted?

Reply:
I removed all streams and add one-by-one